netcdf_to_gltf_converter/config.py

```diff
@@ -180,8 +210,8 @@ class Config(AbstractJsonConfigFile, AbstractFileVersionFile):
     times_per_frame: int
     """int: The number of time steps per animation frame."""
 
-    shift_coordinates: bool
-    """bool: Whether or not to shift the x- and y-coordinates, such that the smallest x and y become the origin (0,0)."""
+    shift_coordinates: Optional[Union[str, CrsShifting]]
+    """Optional[Union[str, CrsShifting]]: The options how to shift the x- and y-coordinates. Typically used to create a reference point, such that an x and y become the origin (0,0)."""
```

Comment:
what is the string in this union?

It the name of this property still descriptive enough?

Reply:
no z coord?